kilt/readers/fid/postprocess.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import json
+
+def convert_to_kilt(inputpath, outputpath, datapath):
+    data = []
+    with open(datapath, 'r') as fin:
+        for k, example in enumerate(fin):
+            example = json.loads(example)
+            data.append(example)
+    datadict = {ex['id']:ex for ex in data}
+    outfile = open(outputpath, 'w')    
+    with open(inputpath, 'r') as f:
+        lines = f.readlines()
+        for line in lines:
+            d = {}
+            try:
+                id, answer = line.split('\t')
+            except ValueError:
+                print('error')
+            id = int(id)
+            answer = answer.split('\n')[0]
+            if id in d:
+                print('key already in dict', d[id], answer)
+            d['id'] = id
+
+            d['output'] = [{'answer': answer}]
```

Comment:
here we need to add an additional field 'provenance'
```
 d['output'] = [{'answer': answer, 'provenance':provenance}]
```
The provenance can be copied as it is from the original input file